sdk/communication/AzureCommunicationChat/Source/Models/CreateThreadResult.swift

```diff
@@ -40,9 +40,10 @@ public struct CreateThreadResult: Codable {
 
     public init(
         from createChatThreadResult: CreateChatThreadResult
-    ) {
+    ) throws {
         self
-            .thread = (createChatThreadResult.chatThread != nil) ? Thread(from: createChatThreadResult.chatThread!) :
+            .thread = (createChatThreadResult.chatThread != nil) ?
```

Comment:
I think instead of using the `?:` we can use `if let` here and it will be cleaner and not use the `!` either
```swift
if let chatThread = createChatThreadResult.chatThread {
   self.thread = try Thread(chatThread)
} else {
   self.thread = nil
}

```